scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -220,37 +255,92 @@ class ScalametaParser(input: Input, dialect: Dialect) { parser =>
           next.is[CloseDelim] &&
           next.pos.startLine > curr.pos.endLine
       if (curr.isNot[Trivia] && !isTrailingComma) {
-        parserTokens += curr
-        parserTokenPositions += currPos
-        val sepRegions1 = {
-          if (curr.is[LeftParen]) ')' :: sepRegions
-          else if (curr.is[LeftBracket]) ']' :: sepRegions
+
+        var sepRegions = if (indentedRegion(sepRegionsP) && currentIndent >= 0) {
+          var sepRegionsI = sepRegionsP
```

Comment:
Maybe rename the vars? It's a bit confusing with some many different `sepRegions`

Reply:
renamed them a little